### DISCUSSION TITLE
Discovering my list of PRs at a destination repo from the source repo

### DISCUSSION THREAD

**Discussion by PhilipOakley** [community/community#36905] [Pull Requests]
If I raise a PR requesting that some fix is taken upstream, then that PR is attached to the upstream repo. 

However I'd like to be able to see, when browsing my originating repo, just which PR's I've raised and possibly their status. 

It's easy to lose track of what's happened to PRs in the distributed async world..

How to find the PR's I've raised? If it's already available, it's not obvious.

This also applies to the automation, so that if I've force pushed some changes, then the CI needs to able to report which PR that was relating to (see [GitGitGadget #1134](https://github.com/gitgitgadget/gitgitgadget/issues/1134#issuecomment-1286661222))

**Comment by webstech** [comment]
[This](https://github.com/pulls?q=is%3Aopen+is%3Apr+author%3APhilipOakley+archived%3Afalse+user%3Agitgitgadget) will get your open PRs in `gitgitgadget`.  On the web interface, just clicking on `Pull requests` at the top of the page will list all your open PRs and you can filter from there.  This may help with part of what you want.
<img width="450" alt="image" src="https://user-images.githubusercontent.com/7956947/197249038-2b375144-48e0-48b1-9c37-0e535e6c6358.png">

It would be nice to see PRs related to a repo maybe on the right under packages (source branch/PR#/target repo).

**Reply by PhilipOakley** [reply]
Thanks

> [This](https://github.com/pulls?q=is%3Aopen+is%3Apr+author%3APhilipOakley+archived%3Afalse+user%3Agitgitgadget) will get your open PRs in `gitgitgadget`.

i.e. the free text version `is:open is:pr author:PhilipOakley archived:false user:gitgitgadget` entered into the search box.... [just checking]

If I want PR's I've raised against other repos, which bit should be tweaked? i.e. how to chose what should be in that `user:` field (or other potential fields? There are lots of moving parts(repos) in the Git and Git-for-Windows eco-system so it's easy to forget what was sent where ;-)

**Reply by webstech** [reply]
> If I want PR's I've raised against other repos, which bit should be tweaked?

https://github.com/pulls will get  you all of your PRs against all repos.  That is where the `Pull requests` button in the posted snip goes.  From there you can do some more filtering and save URLs that you want to use.  Not sure the GitHub cli tool provides a way to extract this info.   Hmmm, could a graphql query extract the info?  Could be scripted - maybe there is an action to do this already.

**Comment by ossanna16** [comment]
Hey there @PhilipOakley and thank you for asking a good question! I'm glad you found our community 🙂

**Reply by PhilipOakley** [reply]
> thank you for asking a good question! I'm glad you found our community

@ossanna16 Thanks go to @dscho for some good pointers !
